synapse/models/gov.py

```diff
@@ -66,7 +66,7 @@
                 ),
                 'doc': 'A US Social Security Number (SSN).'}),
 
-            ('gov:us:zip', ('int', {'regex': '^[0-9]{5}'}), {
```

Comment:
Added a note to the 2x-3x model mapping to figure out if this is supposed to be an int with min/max or str with regex because int with regex isn't a thing.